Src/WitsmlExplorer.Frontend/hooks/useExport.ts

```diff
@@ -0,0 +1,41 @@
+export interface ExportOptions {
```

Comment:
as only one property is optional, "ExportOptions" is not spot on. Perhaps ExportProperties?

Reply:
okay, I renamed it